blockchain.go

```diff
@@ -417,7 +417,10 @@ func configureFVM(conf config, blocks *blocks) (*fvm.VirtualMachine, fvm.Context
 	}
 
 	if !conf.TransactionValidationEnabled {
-		fvmOptions = append(fvmOptions, fvm.WithTransactionProcessors(fvm.NewTransactionInvoker()))
+		fvmOptions = append(
+			fvmOptions,
+			fvm.WithAuthorizationChecksEnabled(false),
```

Comment:
Should we also add `WithAccountKeyWeightThreshold` with negative value to skip signature verification? cc @bluesign